Source/WTF/wtf/Int128.cpp

```diff
@@ -265,9 +265,9 @@ std::ostream& operator<<(std::ostream& os, Int128Impl v) {
       (flags & std::ios::basefield) == std::ios_base::fmtflags();
   if (print_as_decimal) {
     if (Int128High64(v) < 0) {
-      rep = "-";
+      rep.append("-");
     } else if (flags & std::ios::showpos) {
-      rep = "+";
+      rep.append("+");
```

Comment:
Is this GCC 12 warning, correct, or a bug? If it’s a bug, do you have a link to the GCC bug report?

This doesn’t seem like a **great** fix, especially unconditionally on all platforms without any comment. But on the other hand, pretty sure this is dead code in WebKit. We don’t use `Int128` with an `ostream`. So we shouldn’t waste any time on this.

Reply:
I found the bug reported here: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105329

Reported 2022 but still unfixed. I'll add the link to the commit message.

I don't believe there is anything wrong with our code at all. But I don't know if GCC is doing the right thing.

Reply:
Oh it was already merged, well it's documented here :)